doc/LOGGING.md

```diff
@@ -42,6 +42,17 @@ To enable more verbose logging for the AWS Common Runtime that Mountpoint uses t
 
 For finer-grained control over log verbosity, Mountpoint uses the `MOUNTPOINT_LOG` environment variable, which overrides the verbosity options above. The `MOUNTPOINT_LOG` environment variable uses the [`tracing-subscriber` directive syntax](https://docs.rs/tracing-subscriber/0.3.17/tracing_subscriber/filter/struct.EnvFilter.html), and can be used to control log verbosity on a per-subject basis. For example, setting `MOUNTPOINT_LOG` to `trace` enables all trace-level logs, while `trace,awscrt=warn` enables trace-level logs for all log subjects except `awscrt`, which has only warning-level logging enabled.
 
+#### Changing logging verbosity on runtime
+
+Mountpoint allows changing logging verbosity dynamically on runtime using `USR2` Unix signal, e.g., `kill -USR2 <mount-s3-pid>`.
+
+Mountpoint toggles between the following verbosity levels each time it receives a `USR2` signal:
+  1. Default logging verbosity
```

Comment:
Should we add a 'whatever logging level we were initially started with'? I'm not entirely sure if that's what's already implemented, but if so this needs re-wording